tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>@bilgeyucel here it is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>